src/Traits/FactoryTrait.php

```diff
@@ -144,19 +144,31 @@ public static function minValue()
      */
     public static function create($year = null, $month = null, $day = null, $hour = null, $minute = null, $second = null, $tz = null)
     {
-        $year = ($year === null) ? date('Y') : $year;
-        $month = ($month === null) ? date('n') : $month;
-        $day = ($day === null) ? date('j') : $day;
+        $now = isset(static::$testNow) ? static::$testNow->getTimestamp() : time();
+        $defaults = array_combine(
+            [
+                'year',
+                'month',
+                'day',
+                'hour',
+                'minute',
+                'second',
+            ],
+            explode('-', date('Y-n-j-G-i-s', $now))
```

Comment:
Do we need to take timezones into account here? Lower down this date is reconstructed with the `$tz` property, but this is missing a timezone.